account_payment_mode/views/account_payment_method.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- The object account.payment.method is defined in the account module
+but there are no view for this object in the account module... so we define it
+here. I hate the objects that don't have a view... -->
+    <record id="account_payment_method_form" model="ir.ui.view">
+        <field name="name">account_payment_method.form</field>
+        <field name="model">account.payment.method</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        invisible="active"
+                    />
+                    <group name="main">
+                        <field name="name" />
+                        <field name="code" />
+                        <field name="payment_type" />
+                        <field name="bank_account_required" />
+                        <field name="active" invisible="1" />
```

Comment:
```suggestion
```
Now it is no longer necessary to declare a field as invisible in order to use it in any condition of another field.

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0
https://github.com/odoo/odoo/pull/137031